java/semantickernel-api/src/main/java/com/microsoft/semantickernel/orchestration/SKContext.java

```diff
@@ -60,16 +60,16 @@ SKContext build(
      * @param content value to set
      * @return Context for fluent calls
      */
-    SKContext setVariable(String key, String content);
+    SKContext setVariable(String key, ContextVariable<?> content);
 
     /**
-     * Appends data to the given key
+     * Sets the given variable
      *
-     * @param key key to set
+     * @param key     if null defaults to the "input" key
      * @param content value to set
      * @return Context for fluent calls
      */
-    SKContext appendToVariable(String key, String content);
+    SKContext setVariable(String key, Object content);
```

Comment:
I think I'd deprecate this one. The question this will raise is, "What's the difference between this and the other?" If we're going to retain this, then the javadoc should say that it is a convenience method for the other. But then, it can only deal with well-known types. Seems problematic.